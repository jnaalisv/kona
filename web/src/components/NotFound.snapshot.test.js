import React from 'react';
import NotFound from './NotFound';
import renderer from 'react-test-renderer';

test('NotFound renders some html', () => {
    const component = renderer.create(<NotFound />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});