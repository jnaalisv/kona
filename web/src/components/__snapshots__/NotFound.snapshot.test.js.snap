exports[`test NotFound renders some html 1`] = `
<p>
  Not found:(
</p>
`;
